packages/lesmis/src/components/getPartnerSupportItems.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ *
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledLogo = styled.div`
+  width: 55px;
+  height: 55px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: contain;
+  border: 1px solid #dedee0;
```

Comment:
```suggestion
border: ${props => props.theme.palette.grey['400']};
```
Sorry I forgot to explain this but there is a material-ui theme set up for the project and we try to use it when it makes sense. Those times are really just when using brand colors and a couple of very commonly used colors such as this border one.